src/components/planning/Settings.tsx

```diff
@@ -42,208 +43,78 @@ const ButtonWrap = styled.div`
     margin-left: 12px
   }
 `
-const H4 = styled.h4`
-  margin: 18px 0 4px
-`
-const StyledCheckboxes = styled(Checkboxes)`
-  border: 1px solid ${ color("tint", "level5") }
-`
+
+const DAY_PARTS = [
+  'Maandag',
+  'Maandag avond',
+  'Dinsdag',
+  'Dinsdag avond',
+  'Woensdag',
+  'Woensdag avond',
+  'Donderdag',
+  'Donderdag avond',
+  'Vrijdag',
+  'Vrijdag avond',
+  'Weekend',
+  'Weekend avond'
+]
```

Comment:
Remove 'Weekend' and 'Weekend avond'. Add 'Zaterdag weekend' and 'Zondag weekend'

Reply:
Done